lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -103,6 +103,15 @@ class LighthouseReportViewer {
     const gistId = params.get('gist');
     const psiurl = params.get('psiurl');
     const jsonurl = params.get('jsonurl');
+    const gzip = params.get('gzip') === '1';
+    const hashParams = location.hash ?
+      JSON.parse(Base64.decode(location.hash.substr(1), {gzip})) :
```

Comment:
substring or slice instead? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr

Reply:
all three are the same here.